(#206)mobile account

Keegan-lee opened the pull request:
#### What's this PR do?
Creates the MobileAccount.vue component.
Creates the Deposit and Withdraw buttons on the MobileAccount page.

#### Where should the reviewer start?
MobileAccount.vue

#### How should this be manually tested?
Expand and decrease screen size. Compare against Zeplin.

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
#### URL:
[http://staging-ui.trusty.apasia.tech:8080/branch-(-206)MobileAccount](http://staging-ui.trusty.apasia.tech:8080/branch-(-206)MobileAccount)

Commit:
(#206)MobileAccountPage
- Implement Deposit and Withdraw Buttons
- Implement Dropdown and search display

Commit:
(#206) Mobile Account Addition
- Added the template for the mobile account view
- Added the withdraw and deposit buttons
- Styled the withdraw and deposit buttons
- Styled the balance portion of the account portfolio

Commit:
(#206) Mobile Account Modification
- Removed the AccountBalance.vue component
- scoped the styles in MobileAccount
- Protected a null balance value by replacing with 0

roma219 commented on src/views/Account/MobileAccountHeader.vue:
```diff
@@ -0,0 +1,65 @@
+export default {
+  name: 'AccountHeader',
+  components: { Button },
+  props: {
```
these props are not used, aren't they?

roma219 commented on src/views/Account/MobileAccountHeader.vue:
```diff
@@ -0,0 +1,65 @@
+  },
+  computed: {
+    ...mapGetters({
+      historyLoaded: 'history/initialLoaded'
```
not used?

roma219 commented on src/views/Mobile/Mobile.vue:
```diff
@@ -4,17 +4,18 @@
+            height="24"
+            class="search-icon"
+          />
+          <Dropdown
```
Please remove Dropdown for now... Gotta deal with it in a separate issue.

roma219 reviewed (changes requested):
- `MobileAccount` should contain `Portfolio` component - no need to copy-paste all the code.
- Please use rems instead of pixels
- Please don't use hard-code width e.g. `150px` - see comment for details

Structure of `MobileAccount`:
  -- Deposit and Widthdraw buttons (no need for a separate component with them)
  -- Balance
  -- `Portfolio.vue` component

Commit:
#206 Mobile Account Refactor
- Got rid of externalized deposit and withdraw component
- Got rid of unused code
- Got rid of dropdown menu

Keegan-lee committed:
Merge branch 'dev' into (#206)MobileAccount

Keegan-lee committed:
Merge branch 'dev' into (#206)MobileAccount

Commit:
#206
- Changed px to rem
- Moved AccountHeader into MobileAccount
- Move Portfolio into MobileAccount

Commit:
Merge branch '(#206)MobileAccount' of github.com:bitshares/bitshares-community-ui into (#206)MobileAccount

Commit:
#206
- Fixed merge errors

Commit:
#206
- Remove code that is unused in the MobileAccount Component

Commit:
#206 Remove LoadingContainer
- Removed unnessessary LoadingContainer from MobileAccount
